deploy/charts/cert-manager/README.template.md

```diff
@@ -152,6 +152,20 @@ The following table lists the configurable parameters of the cert-manager chart
 | `webhook.securityContext` | Security context for webhook pod assignment | `{}` |
 | `webhook.containerSecurityContext` | Security context to be set on the webhook component container | `{}` |
 | `webhook.hostNetwork` | If `true`, run the Webhook on the host network. | `false` |
+| `webhook.livenessProbe.path` | The liveness probe path | `/healthz` |
+| `webhook.livenessProbe.port` | The livneness probe port | `6080` |
```

Comment:
Would you want to override these, the values are dictated by what the container does, so probing a different path isn't going to work.